src/main/java/org/codehaus/mojo/versions/AbstractVersionsReport.java

```diff
@@ -64,16 +65,16 @@
      *
      * @since 1.0-alpha-3
      */
-    @Component
+    @Inject
```

Comment:
Why here injecting by fields - not by constructor?

Reply:
Oops, fixing it!

Reply:
Thanks for spotting this. Corrected.